compiler/damlc/daml-stdlib-src/DA/Internal/Template/Functions.daml

```diff
@@ -242,6 +248,21 @@ instance Eq AnyContractKey where
   AnyContractKey aAny aRep == AnyContractKey bAny bRep =
     eqAny aAny bAny && aRep == bRep
 
+
+instance Ord AnyTemplate where
+  AnyTemplate a <= AnyTemplate b = lteAny a b
+
+instance Ord AnyChoice where
+  (AnyChoice aAny aRep) <= (AnyChoice bAny bRep)
+    | aRep < bRep = True
+    | aRep == bRep = lteAny aAny bAny
+    | otherwise = False
```

Comment:
Same here. This is a different order (the choice any is compared after the template type rep) but it makes more sense to do this if we're going to use these in maps.

```suggestion
  (<=) = primitive @"BELessEq"
```

Reply:
64cd504949da6ca0d38ce46ee97ab78d92bf2b30